lib/temporal/client.rb

```diff
@@ -132,14 +132,13 @@ def schedule_workflow(workflow, cron_schedule, *input, options: {}, **args)
     #
     # @param name [String] name of the new namespace
     # @param description [String] optional namespace description
-    # @param global [Boolean] flag to set if the namespace is a global namespace or not
-    # @param retention_period [Int] optional retention period in days
-    # @param namespace_data [Hash] A key-value map for any customized purpose
-    def register_namespace(name, description = nil, global: false, retention_period:  10, namespace_data: nil)
-      connection.register_namespace(name: name, description: description, global: global, retention_period: retention_period, namespace_data: namespace_data)
+    # @param is_global [Boolean] used to distinguish local namespaces from global namespaces (https://docs.temporal.io/docs/server/namespaces/#global-namespaces)
+    # @param retention_period_days [Int] optional  value which specifies how long Temporal will keep workflows after completing
+    # @param namespace_data [Hash] optional key-value map for any customized purpose that can be retreived with describe_namespace
+    def register_namespace(name, description = nil, is_global: false, retention_period_days:  10, data: nil)
+      connection.register_namespace(name: name, description: description, is_global: is_global, retention_period_days: retention_period_days, data: data)
```

Comment:
By changing `global` to `is_global` this is a breaking change. Is the rename really worth it? I'd say no.

Same for the other arguments `retention_period` and `namespace_data`. 

I think this is best handled by improving the documentation on the header as you have done. The argument changes will cause breakage on any upgrade and I'm not convinced the new names are that much better.

@antstorm opinion?

Reply:
Hey, no API changed here (so no breaking change). I think the confusion is previously in this PR this param was called `global` but before this PR, `global` never existed here! Happy to chat if it's something else?

Reply:
Yes, there should be no breaking changes since GRPC interface is private. As for adding `_days` I think this needs to be addressed across more broadly (if this is the pattern we'd like to follow) and not individually